package.json

```diff
@@ -1,10 +1,7 @@
 {
   "name": "cozy-ui",
-  "version": "0.1.7",
+  "version": "3.0.0",
   "description": "Cozy apps Ui SDK",
-  "scripts": {
```

Comment:
@GoOz It is intented to be used as a stylus plugin. It's the project that will include cozy-ui that will build the CSS from stylus. On npm and github, cozy-ui will be just a set of stylus files.